demisto_sdk/commands/common/logger.py

```diff
@@ -63,6 +47,90 @@ def parse_int_or_default(value: Any, default: int) -> int:
     os.getenv("DEMISTO_SDK_LOG_FILE_COUNT"), 10
 )
 
+FILE_LOG_RECORD_FORMAT = "[%(asctime)s] - [%(threadName)s] - [%(levelname)s] - %(filename)s:%(lineno)d - %(message)s"
+
+if os.getenv("CI"):
+    CONSOLE_LOG_RECORD_FORMAT = "[%(asctime)s] [%(levelname)s] %(message)s"
+    CONSOLE_LOG_RECORD_FORMAT_SHORT = "[%(asctime)s] [%(levelname)s] "
+else:
+    CONSOLE_LOG_RECORD_FORMAT = "[%(levelname)s] %(message)s"
+    CONSOLE_LOG_RECORD_FORMAT_SHORT = "[%(levelname)s] "
+
+
+CONSOLE_RECORD_FORMATS = {
+    logging.DEBUG: "[lightgrey]%(message)s[/lightgrey]",
+    logging.INFO: "[lightgrey]%(message)s[/lightgrey]",
+    logging.WARNING: "[yellow]%(message)s[/yellow]",
+    logging.ERROR: "[red]%(message)s[/red]",
+    logging.CRITICAL: "[red][bold]%(message)s[/bold[/red]",
+    SUCCESS_LEVEL: "[green]%(message)s[/green]",
+}
+
+NO_COLOR_ESCAPE_CHAR = "\033[0m"
+
+DEMISTO_LOG_ALLOWED_ESCAPES = [  # The order of the list is by priority.
+    ("green", 32),
+    ("red", 91),
+    ("yellow", 93),
+    ("cyan", 36),
+    ("blue", 34),
+    ("orange", 33),
+    ("pink", 95),
+    ("purple", 35),
+    ("black", 30),
+    ("invisible", 8),
+    ("bold", 1),
+    ("disable", 2),
+    ("reverse", 7),
+    ("strikethrough", 9),
+    ("underline", 4),
+    ("darkgrey", 90),
+    ("darkred", 31),
+    ("lightblue", 94),
+    ("lightcyan", 96),
+    ("lightgreen", 92),
+    ("lightgrey", 37),
+    ("lightred", 91),
+]
+
+DEMISTO_LOG_LOOKUP = dict(  # Convert the list of tuples to a dict
```

Comment:
can we make it a dict comp, or simplify it in a different way?

Reply:
that's the simplest i could think of. i couldn't find another pattern to create 2 key per item in the list.